backstory/src/main/java/com/google/sps/servlets/data/DatastoreServiceFactorySps.java

```diff
@@ -0,0 +1,29 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets.data;
+
+import com.google.appengine.api.datastore.DatastoreService;
+
+/**
+ * Factory pattern for DatastoreService.
+ */
+public interface DatastoreServiceFactorySps {
```

Comment:
I'm curious what does `Sps` mean? Can the abbreviation by avoided? Can Factory be the last word in this name, to indicate that this is a factory?

Reply:
The reason for this is that there is already a DatastoreServiceFactory needed to create real DatastoreService objects. SPS stands for Software Product Sprint. The SPS is only there to differentiate the two factories.

Reply:
Actually, I think I will call it BackstoryDatastoreServiceFactory to avoid all of these problems.